src/viam_mlmodelservice_triton_impl.cpp

```diff
@@ -338,6 +340,90 @@ class Service : public vsdk::MLModelService, public vsdk::Stoppable, public vsdk
         return state_;
     }
 
+    static std::string initialize_directory_(std::string model_name, std::int64_t version, bool is_tf) {
```

Comment:
- If `model_name` isn't being consumed (and I don't think it is) better style is to pass it by `const &`. The `operator<<` below takes it's argument by `const&` so it can't move it.
- I dislike bool parameters in general, and it looks to me like it could fairly easily be eliminated if this function just took the path to initialize. It feels to me like determining the name under which to place the model belongs more in `symlink_model_` anyway.